sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -154,6 +155,13 @@ def _build_uri(address, entity):
     address += "/" + str(entity)
     return address
 
+def _get_backoff_time(retry_mode, backoff_factor, retried_times):
+    if retry_mode == RetryMode.Fixed:
```

Comment:
@yunhaoling in azure-core, they do a check on `retried_times`  and ["ignore redirects"](https://github.com/Azure/azure-sdk-for-python/blob/35d5436cf58a207613cc2bfd1fa796c67f92af1e/sdk/core/azure-core/azure/core/pipeline/policies/_retry.py#L112). It looks like we don't do this check. Is this b/c we're using amqp here? Or b/c we are checking this already elsewhere?

Reply:
We're checking [reties_time](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py#L540) here.

from core code itself, I think it does backoff starting from the second time when an error (consecutive error) is met.
as far as I know we're not doing redirect in eh/sb, so I think we could ignore "ignore redirect" for now.